src/OMSimulatorLib/Scope.cpp

```diff
@@ -205,6 +205,21 @@ oms3::Model* oms3::Scope::getModel(const oms3::ComRef& cref)
   return models[it->second];
 }
 
+
+oms_status_enu_t oms3::Scope::addConnector(const oms3::ComRef &cref, oms_causality_enu_t causality, oms_signal_type_enu_t type)
+{
+  oms3::ComRef head(cref);
+  oms3::ComRef back = head.pop_back();
```

Comment:
`oms3::ComRef::pop_back` shouldn't be used and probably just removed. A ComRef needs always to be resolved front to back, because a signal name might contain dots.